### ISSUE TITLE
Feature Request: Positional Feedback only on misses.

### ISSUE THREAD

**Issue by steeben** [PunishXIV/Avarice#5]
I like the idea of the "✓" or "X" for positional, but find the "✓" on every successful positional a tad excessive. Would it be possible to add another option to have only the "X" for missed positionals appear, with successful positionals not showing anything?

Excellent plugin none the less!

**Comment by kawaii** [comment]
Yeah, I've been meaning to give Avarice some updates recently, I'll make a note of this.

**Comment by Kagekazu** [comment]
that is added as an option
